dev/devicelab/lib/framework/devices.dart

```diff
@@ -534,21 +533,23 @@ class AndroidDevice extends Device {
   }
 
   /// Executes [command] on `adb shell` and returns its exit code.
-  Future<void> shellExec(String command, List<String> arguments, { Map<String, String>? environment, bool silent = false }) async {
+  Future<void> shellExec(String command, List<String> arguments,
+      {Map<String, String>? environment, bool silent = false}) async {
```

Comment:
Maybe another trailing comma after `silent = false`, and below?

Reply:
Reverted back to previous formatting to simplify the code review